app/assets/javascripts/preferences/panes/two-factor-auth/TwoFactorAuth.ts

```diff
@@ -11,7 +11,7 @@ export const is2FADisabled = (s: TwoFactorStatus): s is 'two-factor-disabled' =>
   s === 'two-factor-disabled';
 
 export const is2FAActivation = (s: TwoFactorStatus): s is TwoFactorActivation =>
```

Comment:
Let's use more meaningful name for `s` :)

Reply:
Done.